docs/pages/blog/lab-pickers-to-mui-x.md

```diff
@@ -0,0 +1,63 @@
+---
+title: The date pickers are now part of the Advanced components
```

Comment:
Yeah the current title is very factual, not catchy at all

Reply:
What details deserve to be highlighted in the title and description? 

I think the most interesting thing we could convey to the reader is "these components are going to become more powerful and sophisticated now that they're a part of MUI X."

In other words, you will have even more powerful tools at your disposal if you opt for MUI X and/or the Pro plan.

Ultimately, we want to persuade developers to try MUI X and purchase a commercial license. So we need to use the headline and description to present the primary value proposition.

I'm not in love with this suggestion—it's centered on "us" rather than the reader—but I think it's at least a step in the right direction.

```suggestion
title: MUI X expands its product line with advanced date and time picker components
```

Reply:
I'm putting this title
If somebody have ideas to improve it